src/utils/filter.ts

```diff
@@ -42,6 +42,24 @@ function isMultiPathSupported(
     return expandedPaths.every((expandedPath) => isPathSupported(expandedPath, filterObjects));
 }
 
+function setDefaultAsFirst(
+    pathObjects: PathObject[],
+    filterObjects: FilterObject[],
+): PathObject[] {
+    const defaultFilter = filterObjects.find((f) => f.isDefault);
+    if (!defaultFilter) return pathObjects;
+
+    const defaultIndex = pathObjects.findIndex((pathObj) => {
+        return isPathSupported(pathObj, [defaultFilter]);
+    });
+    if (defaultIndex < 0 || pathObjects.length < 2) return pathObjects;
+
+    const tmp = pathObjects[1];
```

Comment:
Same as above for the indices

Reply:
corrected the error and added explanatory comment